camera_matrix:
- - 1149.7958276237264
  - 0.0
  - 962.9748949950225
- - 0.0
  - 1148.4015894864226
  - 562.6395918748777
- - 0.0
  - 0.0
  - 1.0
distortion_coefficients:
- - 0.001817991728558115
  - -0.005113244944484045
  - 0.0004743717046200585
  - 0.00024342016144968313
  - 0.0028870851607187887
image_height: 1080
image_width: 1920
